modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -50,10 +51,12 @@
 
 /datum/brain_trauma/special/nymphomania
 	name = "Nymphomania"
-	desc = "The patient constantly feels aroused and supposed to satisfy their sexual desires."
-	scan_desc = "constant sexual arousal"
+
+	desc = "The patient constantly feels sexual arousal and heightened libido."
+	scan_desc = "severly heightened libido"
```

Comment:
Make it not have appear on scans and make sure that it can't be rolled as a random brain trauma via various methods, at the very least.

Reply:
It can't roll, I'll get rid of the scan stuff.